agent/backend/pktvisor/scrape.go

```diff
@@ -156,9 +157,78 @@ func (p *pktvisorBackend) scrapeDefault() error {
 	return nil
 }
 
+func (p *pktvisorBackend) scrapeOtlp() {
```

Comment:
this opens up a port as a server, right? let's not call it "scrape" which implies that this will connect outwards, which it won't. maybe 'receiveOtlp'

Reply:
yes, you are right. done